hack/e2e-scheduling-cluster.sh

```
@@ -0,0 +1,136 @@
+#   KIND_CLUSTER_NAME     — Kind cluster name (default: was-test)
+#   K8S_MAIN_NODE_IMAGE   — Kind node image name (default: k8s-main:latest)
+#   IMAGE_TAG             — JobSet controller image tag
+#   ARTIFACTS             — directory for logs and test artifacts
+#   E2E_TARGET_FOLDER     — kustomize config folder (default: scheduling)
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Resolve tool paths to absolute to survive subshell cd operations.
+export KUSTOMIZE="$(cd "$PWD" && realpath "${KUSTOMIZE:-$PWD/bin/kustomize}")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)